api/v1beta1/hyperconverged_types.go

```diff
@@ -430,13 +430,6 @@ type HyperConvergedFeatureGates struct {
 	// +default=false
 	DeployKubeSecondaryDNS *bool `json:"deployKubeSecondaryDNS,omitempty"`
 
-	// Deploy KubevirtIpamController by CNAO.
-	// Allows having persistent IPs for Kubevirt user defined networks.
-	// +optional
-	// +kubebuilder:default=false
-	// +default=false
-	DeployKubevirtIpamController *bool `json:"deployKubevirtIpamController,omitempty"`
-
```

Comment:
According to this https://kubernetes.io/docs/reference/using-api/deprecation-policy/#deprecating-parts-of-the-api, we can't just drop API fields. We'll do it in v1 of the API.

All the rest of the code is great, e.g. removing the default value etc.

Please just deprecate it, but don't remove it from the API

```go
    // Deprecated: this field is ignored and will be removed in the next version of the API.
	DeployKubevirtIpamController *bool `json:"deployKubevirtIpamController,omitempty"`

```

Reply:
Should I remove it from the docs ? @nunnatsa

Reply:
I think it's ok to remove it from the doc. it is ignored anyway. 

Also, there is a mistake in my previous comment. the `semverRange` in the json I wrote should be `<=1.4.0`, not `<1.14.0`

Reply:
Done. Thank you.